packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -315,6 +372,8 @@ export class GraphQLApi extends Construct {
   public readonly arn: string;
   /**
    * the URL of the endpoint created by AppSync
+   *
+   * @attribute
    */
   public readonly graphQlUrl: string;
```

Comment:
curious about the casing here (and everywhere really)... how does this translate to Python where we do snake casing?

does `GraphQL` get broken down into `graph_q_l`?

Reply:
yeah i think in this case would be best to change all of them to graphqlUrl for props

Reply:
^ this would be a breaking change tho so not sure if thats the best move for something so small..

@MrArnoldPalmer  thoughts?

Reply:
I'm not sure it needs to happen in this PR, but fixing the casing sounds like something we need to do regardless if it requires code that is not idiomatic for Python developers

Reply:
yeah i think a chore might be better